jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jakarta-tests/src/test/java/org/eclipse/jetty/ee10/websocket/jakarta/tests/client/WriteTimeoutTest.java

```diff
@@ -89,14 +90,15 @@ public void testTimeoutFromSlowReads() throws Exception
     {
         EventSocket clientEndpoint = new EventSocket();
         Session session = client.connectToServer(clientEndpoint, server.getWsUri());
-        session.getAsyncRemote().setSendTimeout(1000);
+        RemoteEndpoint.Async asyncRemote = session.getAsyncRemote();
```

Comment:
Fix this test too, like the core one: remove the static `CountDownLatch`, etc.

Reply:
I can do this for the core version, but not sure how to do it in the Jakarta API. 

To register the server endpoint you use the class not an instance, so I don't think there is a way to do it without the static.

Reply:
You can, with this (a sketch, but you get the idea):

```java
start(Supplier<ServerSocket> endPointSupplier) 
{
    ...
            server.getServerContainer().addEndpoint(ServerEndpointConfig.Builder.create(ServerSocket.class, "/")
            .configurator(new ServerEndpointConfig.Configurator()
            {
                @Override
                public <T> T getEndpointInstance(Class<T> endpointClass) throws InstantiationException
                {
                    return endPointSupplier.get();
                }
            }).build());
    ...
}
```